src/main/java/pl/Lukasz/Calories/Repository/AuthenticatingRepository.java

```diff
@@ -0,0 +1,11 @@
+package pl.Lukasz.Calories.Repository;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import pl.Lukasz.Calories.Entity.Users;
+
+public interface AuthenticatingRepository extends CrudRepository <Users, Integer> {
+@Query(value = "select * from users where username= ?1", nativeQuery = true)
```

Comment:
code formatting
also, why native query? You can use special method name from spring security, or write a HQL, but I guess it can also be native - as an exercise try use other methods as well